### DISCUSSION TITLE
Can someone take a look at the yaml and see why using groups isn't working.

### DISCUSSION THREAD

**Discussion by gorrilla10101** [caronc/apprise-api#144] [General]
I was using the apprise ui to configure and test. if I call the tags directly this configuration works however `POST /notify/recipes` does not work. Instead it returns `424 (Failed Dependency)`. 

I didn't see anything more meaningful in the api logs than I got from the http response. 
```log
2023-11-07 23:19:01,290 [WARNING] django: APPRISE_DENY_SERVICES plugin macos:// was not found - ignoring.
2023-11-07 23:19:01,304 [INFO] apprise: Loaded 2 entries from memory://
2023-11-07 23:19:01,307 [WARNING] django.request: Failed Dependency: /notify/recipes127.0.0.1 - - [08/Nov/2023:05:19:01 +0000] 
"POST /notify/recipes HTTP/1.1" 424 190 "http://localhost:8000/cfg/recipes" 
"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36 Edg/119.0.0.0"
```

This is my yaml file

```yaml
version: 1
groups:
  recipes: user1, user2
  
urls:
  - mailgun:///noreply@mg.example.com/{token}
    - to: user1@example.com
      tag: user1
    - to: user2@example.com
      tag: user2
```

**Comment by caronc** [comment]
The good news is that this isn't a bug, but more of an enhancement.  I didn't realize i didn't support the structure you provided and have since added support [here](https://github.com/caronc/apprise/pull/998).

But in the meantime, you can still use the current version to do your bidding, just change your `groups` entry into a list for now:
```yaml
version: 1
groups:
  # only difference is i added a `-` to turn the entry below into a list
  # stay tune while i make the way you had it work (future)
  - recipes: user1, user2
  
urls:
  - mailgun:///noreply@mg.example.com/{token}:
    - to: user1@example.com
      tag: user1
    - to: user2@example.com
      tag: user2

**Reply by caronc** [reply]
@gorrilla10101 : your config will now work in the latest version of Apprise in addition to the one i shared above.

**Reply by jeffreytyler** [reply]
Thanks, I missed the original notification. I'm away at the moment but will give it a shot when in a couple days.

**Reply by gorrilla10101** [reply]
That works, thanks.
